.circleci/config.yml

```diff
@@ -7,6 +7,8 @@ orbs:
   # be found at http://github.com/apollographql/CircleCI-Orbs/.
   # We could use Renovate to bump this version via PR, but that's not set up now.
   oss: apollo/oss-ci-cd-tooling@0.0.16
+  rust: circleci/rust@1.5.0
+  win: circleci/windows@2.3.0
```

Comment:
BTW I think this might be unused?

Reply:
Poked around a bit and this seems correct, pushed 997dfee and will see how CI does.

Reply:
Good catch!  Indeed not necessary.  I had started with this approach, but I couldn't customize one thing I needed to within that orb and didn't have time to draw up a contribution to the orb itself.